view/activity_activity_tree.xml

```diff
@@ -11,6 +11,7 @@ contains the full copyright notices and license terms. -->
     <field name="party" expand="1"/>
     <field name="resource" expand="2"/>
     <field name="state"/>
+    <field name="mine" optional="1"/>
```

Comment:
I think it should be invisible. After all, the employee field already shows if the activity is mine or not.